SimTKcommon/Simulation/src/State.cpp

```diff
@@ -312,7 +313,10 @@ StateImpl::StateImpl(const StateImpl& src)
 :   currentSystemStage(Stage::Empty)
 {
     initializeStageVersions();
-
+    //TODO: When you copy a state, you don't get the same mutex, is that okay? 
```

Comment:
Yes, that's perfect. Every State object should have its own unique mutex. You can delete this comment.